README.md

```diff
@@ -16,7 +16,7 @@ I have done the first one for you, as an example.
 | $f(n)=n$                   | 1           | $g(n)=2n^2 + n$        |
 | $f(n)= 10n + 3\log_{15} n$ | 4            | $g(n)= 4n - 2\log_2 n$ |
 | $f(n) = 2n^5$              | 2            | $g(n) = 5n^2$          |
-| $f(n)=\log_{10} \left(n^{10}\right)$ | 4 | $g(n)=n$ |
+| $f(n)=\log_{10} \left(n^{10}\right)$ | 1 | $g(n)=n$ |
 | $f(n)= 4n^5 $ | 2 | $g(n)= 5n^4$ |
 | $f(n) = 10^{256}$ | 3 | $g(n) = \log n$ |
```

Comment:
The one before the last is incorrect! log n has a variable n, which could be way bigger asymptotically for large n's, compared to 10^256.